mod/wikirate/set/right/image.rb

```diff
@@ -3,10 +3,14 @@
   view :missing do |args|
     # try to get type+missing_image_card's image to display
     # EX: user+missing_image_card
-    parent_type_card = Card[card.left.type_id]
-    missing_image_card = parent_type_card.fetch(:trait=>:missing_image_card)
-    if missing_image_card
-      subformat( missing_image_card )._render args[:home_view],args
+    if card.left.present?
+      parent_type_card = Card[card.left.type_id]
+      missing_image_card = parent_type_card.fetch(:trait=>:missing_image_card)
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.